jarviscli/plugins/stt.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+import speech_recognition as sr
+import pyttsx3
+import os
+from plugin import plugin
+import Jarvis
+
+@plugin()
+def hear(jarvis, s):
+    r = sr.Recognizer()
+    engine = pyttsx3.init()
+    listen = False
+
+    while listen == False:
+        try:
+            with sr.Microphone() as source:
+                os.system("reset")
+                print("Say listen to start listening")
+                r.adjust_for_ambient_noise(source)
+                audio = r.listen(source)
+                pinger=r.recognize_google(audio)
+            try:
+                if (pinger.lower() == "listen") :
+                    listen = True
+                    print("Voice mode activated. Say something!")
+                    break
+                else:
+                    continue
+            except LookupError:
+                continue
+        except sr.UnknownValueError:
+            continue
+
+    while listen == True:
+        print ("Say somthing")
+        try:
+            with sr.Microphone() as source:
+                r.adjust_for_ambient_noise(source)
+                audio = r.listen(source)
+                pinger=r.recognize_google(audio)
+
+            if (pinger.lower() == "stop") :
+                listen = False
+                print("Listening stopped.")
+                break
+            else:
+                print (pinger)
+                if pinger[0:5] == "go to":
+                    os.system("wmctrl -a " + pinger[7:])
+                elif pinger[0:9] == "workspace":
+                    pinger = pinger[11:len(pinger)]
+                    if pinger == 'one':
+                        pinger = 1
+                    os.system("wmctrl -s " + str(int(voice) - 1))
+                else:
+                    line = pinger
+                    jarvis = Jarvis.Jarvis()
```

Comment:
Just noticed, no need to create new Jarvis instance here - ``jarvis._jarvis`` should also work. Or we just add a ``eval(str)``-method to JarvisAPI.

Reply:
Done.